### ISSUE TITLE
There ought to be a zoom-in and zoom-out button within the "colorMyWorld" activity.

### ISSUE THREAD

**Issue by d1vyanshu-kumar** [llaske/sugarizer#1593]
@llaske I think there should be a zoom-in and zoom-out button in the activity colorMyWorld.

<img width="87" alt="Screenshot 2024-04-10 at 2 51 24 PM" src="https://github.com/llaske/sugarizer/assets/154233802/c2d1f9df-c3c3-4c7d-925d-80d507af1724">

**Comment by d1vyanshu-kumar** [comment]
@llaske please review my issue if this looks genuine then I would like to work on this issue.

**Comment by d1vyanshu-kumar** [comment]
@llaske can I start work on this?

**Comment by llaske** [comment]
Could be a good idea.

**Comment by SamarthBagga** [comment]
@llaske , I was thinking of adding the zoom in and out buttons to the toolbar. Is this fine or should these buttons be like the ones in the issue description?

**Comment by llaske** [comment]
@SamarthBagga yes you could reuse the same zoom palette that the one in Calligra activity

![image](https://github.com/llaske/sugarizer/assets/981841/839551ae-728f-4aed-beb6-423032ec3eb5)

**Comment by SamarthBagga** [comment]
Okay, I'll start work on this

**Comment by llaske** [comment]
Fixed in https://github.com/llaske/sugarizer/pull/1602
